vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -784,9 +784,22 @@ private void addWorkflowVersion(HttpServerExchange exchange, AddWorkflowVersionR
           .transaction(
               context -> {
                 final var dsl = DSL.using(context);
+                DSL.using(connection, SQLDialect.POSTGRES)
+                    .select(WORKFLOW.ID)
+                    .from(WORKFLOW)
+                    .where(WORKFLOW.NAME.eq(name))
+                    .fetchOptional(Record1::value1)
+                    .ifPresentOrElse(
+                        s -> System.out.println(""),
+                        () -> {
+                          exchange.setStatusCode(StatusCodes.NOT_FOUND);
+                          exchange
+                              .getResponseSender()
+                              .send(String.format("No workflow with name %s found", name));
+                        });
```

Comment:
This doesn't actually bail out on this flow though.

Reply:
Sigh you're right. That commit really wasn't ready for primetime. I've updated it, now with working-as-expected code...